js/ui/form/components/field_item.js

```diff
@@ -77,7 +77,9 @@ export function renderFieldItem({
     let $label = null;
 
     if(needRenderLabel) {
-        labelOptions.labelTemplateData = getTemplateData(item, editorOptions, formOrLayoutManager);
+        if(labelOptions.labelTemplate) {
+            labelOptions.labelTemplateData = getTemplateData(item, editorOptions, formOrLayoutManager);
```

Comment:
We can write it without condition
`labelOptions.labelTemplateData ??= getTemplateData(item, editorOptions, formOrLayoutManager);`